nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -563,6 +563,12 @@ can now be used to disable the upstream fallback servers entirely by setting it
 
 - `virtialisation.incus` now defaults to the newly-added `incus-lts` release (v6.0.x). Users who wish to continue using the non-LTS release will need to set `virtualisation.incus.package = pkgs.incus`. Stable release users are encouraged to stay on the LTS release as non-LTS releases will by default not be backported.
 
+- Move `clang-tools` into `llvmPackages`.
```

Comment:
Addressed. Thank you for pointing out.

By the way, I once received a review from @fricklerhandwerk in another PR that suggests using imperative tones when writing release notes. However, most of the entries in this edited release note use passive tones so that the target of changes becomes the first word of a sentence.

Do you think I should change the newly added entry and use a passive tone instead?

Reply:
As far as I know indicative mood with the subject up-front is the aim here but @fricklerhandwerk likely knows better than I do; I don't interact with the release notes much 🙂 

cc: @jacg

Reply:
TLDR: yes, please change the entry to match the style used throughout the rest of document.

For the 24.05 release, I was instructed to ensure that the name of the affected item appear at or near the beginning of every note (in order to make it easier to spot relevant entries when scanning the document), hence the subject-up-front indicative style is used throughout the document. This was already the prevailing style before I started editing the notes.

I would therefore argue that, in this context, all notes should follow that style, regardless of what merits other styles might have in other contexts.